repository/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -83,6 +83,7 @@
     <include file="db/changelog/db.changelog-rdm-7575.xml"/>
     <include file="db/changelog/db.changelog-rdm-7575-cleanup.xml"/>
     <include file="db/changelog/db.changelog-rdm-8223.xml"/>
+    <include file="db/changelog/db.changelog-rdm-8382.xml"/>
```

Comment:
This should be listed as the final migration otherwise will cause errors when merged?